Brokkr/src/main/include/commands/IntakeSpin.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <frc2/command/CommandBase.h>
+#include <frc2/command/CommandHelper.h>
+#include "subsystems/Claw.h"
+#include <frc/Timer.h>
+
+/**
+ * An example command.
+ *
+ * <p>Note that this extends CommandHelper, rather extending CommandBase
+ * directly; this is crucially important, or else the decorator functions in
+ * Command will *not* work!
+ */
+class IntakeSpin
+    : public frc2::CommandHelper<frc2::CommandBase, IntakeSpin> {
+ public:
+  IntakeSpin(Claw& mClaw);
```

Comment:
Name parameters with all lower case. Member variables are prefixed with `m`. (This is not a member variable.)

`  IntakeSpin(Claw& claw);`